rj_gameplay/rj_gameplay/play/basic_defense.py

```diff
@@ -30,14 +30,20 @@ def tick(
         world_state: stp.rc.WorldState,
     ) -> List[RobotIntent]:
 
-        if self.state == "init":
-            self.prioritized_tactics.append(goalie_tactic.GoalieTactic(world_state, 0))
-            self.prioritized_tactics.append(wall_tactic.WallTactic(world_state, 5))
-            # TODO: add nmark tactic
-            #       and make it go for the ball (rather than stopping in front)
-            self.assign_roles(world_state)
-            self.state = "active"
-            return self.get_robot_intents(world_state)
-        elif self.state == "active":
+        if self.state is "init":
+            # TODO: had to add this check or role assignment behaved oddly
+            #       fix by updating gameplay node to only tick once world_state is not None
+            if world_state is not None:
+                self.prioritized_tactics.append(
+                    goalie_tactic.GoalieTactic(world_state, 0)
+                )
+                self.prioritized_tactics.append(wall_tactic.WallTactic(world_state, 3))
+                # TODO: add nmark tactic
+                #       and make it go for the ball (rather than stopping in front)
+                self.prioritized_tactics.append(nmark_tactic.NMarkTactic(world_state, 2))
```

Comment:
you never imported the nmark_tactic at the top, so style is complaining (it still works bc Python)